### ISSUE TITLE
Improve PyOpenMS Tutorial

### ISSUE THREAD

**Issue by oliveralka** [OpenMS/Tutorials#190]
- [ ] Add a sentence or two regarding the Binder integration. 
- [ ] Change all the links to the readthedocs to "latest" instead of pointing to the 2.4.0

**Comment by timosachsenberg** [comment]
we could also move missing parts into the readthedocs page.
IMHO we would reduce the number of tutorials we need to maintain

**Comment by timosachsenberg** [comment]
@oliveralka and @jpfeuffer what do you think?

**Comment by jpfeuffer** [comment]
Can you elaborate on your comment?

**Comment by timosachsenberg** [comment]
remove from this github and move content on readthedocs page

**Comment by jpfeuffer** [comment]
Yes sure. Maybe leave the section and say. This documentation has moved to:...
